api/android/build-android-lib.sh

```diff
@@ -40,10 +40,10 @@ for arg in "$@"; do
             ;;
         --target_abi=*)
             target_abi=${arg#*=}
-            if [[ $target_abi == 'arm64' ]]; then
-                nnstreamer_target_abi="'arm64-v8a'"
-            elif [[ $target_abi == 'armv7' ]]; then
+            if [[ $target_abi == 'armv7' ]]; then
                 nnstreamer_target_abi="'armeabi-v7a'"
+            elif [[ $target_abi == 'armv8' ]]; then
+                nnstreamer_target_abi="'arm64-v8a'"
```

Comment:
armv8-64bit == aarch64
armv8-32bit == aarch32

This is why they usually don't use the term "armv8" for binaries in build systems.

For example, Cortex-A32 is 32bit system while it is armv8.

Reply:
ok, I will update to arm64 (rollback)